src/segment_language.go

```diff
@@ -43,6 +57,20 @@ func (l *language) string() string {
 		return l.props.getString(MissingCommandTextProperty, MissingCommandText)
 	}
 	l.setVersion()
+
+	// if regex found no version
+	if l.version == "" {
+		return ""
+	}
+
+	// build release notes hyperlink
+	if l.props.getBool(EnableHyperlink, false) {
```

Comment:
We might want to validate the presence of a `urlTemplate` too

Reply:
Yes, in the end if it's blank it will return the version(error will be returned from TruncatingSprintf. But an extra check before calling it would be better.